libp2p/multicodec.nim

```diff
@@ -251,37 +257,32 @@ const
 
 proc multiCodec*(name: string): MultiCodec {.compileTime.} =
   ## Generate MultiCodec from string ``name`` at compile time.
-  var code = NameCodecs.getOrDefault(name, -1)
-  if code == -1:
-    raise newException(MultiCodecError,
-                       "MultiCodec `" & name & "` not supported!")
-  result = MultiCodec(code)
+  let code = NameCodecs.getOrDefault(name, -1)
+  doAssert(code != -1)
```

Comment:
This is COMPILE TIME code, asserting is fine as gives complete __LINE__ and __FILE__